draft-iab-covid19-workshop.md

```diff
@@ -273,25 +273,27 @@ impact on networking. With large numbers of people working from home
 or otherwise depending on the network for their daily lives, network
 traffic volume has surged. Internet service providers and operators have
 reported a 20% traffic growth or more in a matter of weeks. Traffic at
-Internet Exchange Points (IXPs) is similarly on the rise. Most forms
+Internet Exchange Points (IXPs) is similarly on the rise, as were attacks targeting remote working infrastructure and platforms. Most forms
 of network traffic have seen an increase, with conversational
-multimedia traffic growing in some cases more than 200%. And user time
+multimedia traffic growing in some cases more than 200% and an explosion in COVID-19 related scams and phishing. And user time
```

Comment:
Same here. Let's rather at a separate paragraph (please provide a separate PR if possible). video traffic is really only meant to be one example where we have a high number.